docker/docker_client.go

```diff
@@ -361,7 +424,7 @@ func SearchRegistry(ctx context.Context, sys *types.SystemContext, registry, ima
 		q.Set("n", strconv.Itoa(limit))
 		u.RawQuery = q.Encode()
 
-		logrus.Debugf("trying to talk to v1 search endpoint\n")
+		logrus.Debugf("trying to talk to v1 search endpoint")
```

Comment:
Such cleanups are welcome, but please make them in a separate commit, if not a separate PR.